.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "HiGHS"]
 	path = HiGHS
-	url = git@github.com:ERGO-Code/HiGHS.git
+	url = https://github.com/ERGO-Code/HiGHS.git
```

Comment:
why ?

Reply:
Will be removed on next update - our CI pipeline was having trouble downloading for some reason...